.travis.yml

```diff
@@ -1,39 +1,29 @@
+if: type = pull_request
```

Comment:
I had removed this recently since it broke the build badge in the readme (or more generally, it did not build master).